opentelekomcloud/services/obs/resource_opentelekomcloud_obs_bucket.go

```diff
@@ -1141,14 +1132,14 @@ func GetObsError(action string, bucket string, err error) error {
 
 // normalize format of storage class
 func normalizeStorageClass(class string) string {
-	var ret = class
+	var normalizedClass = class
 
 	if class == "STANDARD_IA" {
-		ret = "WARM"
+		normalizedClass = "WARM"
 	} else if class == "GLACIER" {
-		ret = "COLD"
+		normalizedClass = "COLD"
 	}
-	return ret
+	return normalizedClass
```

Comment:
No need to introduce a variable. You can just return value immediately. And also you can use switch-case